nms/app/packages/magmalte/app/views/equipment/GatewayPoolGatewaysEdit.js

```diff
@@ -133,9 +150,9 @@ export default function GatewayEdit(props: GatewayPoolEditProps) {
                     variant={'outlined'}
                     displayEmpty={true}
                     value={gw.gateway_id}
-                    onChange={({target}) =>
-                      handleGwIdChange(target.value, index)
-                    }
+                    onChange={({target}) => {
+                      handleGwIdChange(target.value, index, gw.gateway_id);
+                    }}
```

Comment:
For this dropdown menu, add a placeholder value of `Unpooled Gateway ID` or other appropriate value to communicate that only gatewayIDs that aren't part of a gateway pool can be selected.

Also, only display the selectable gateway IDs

Reply:
Not sure I understand 😄 
Right now I'm displaying all the gateway ids, but if a gateway id is associated to another gateway pool the option is disabled. So the only gateway IDs that the user can select are gateway IDs that aren't part of a gateway pool.

Reply:
So, rather than disabling some of the options, just not showing them.

https://user-images.githubusercontent.com/804385/114628666-1523c980-9c6c-11eb-820d-3a798114d1a8.mov

So what I mean by 'placeholder value' would correspond to 'None' in the small recording above.